osbuild/__init__.py

```diff
@@ -116,13 +123,25 @@ def run(self, argv, binds=None, readonly_binds=None, **kwargs):
             f"--machine={self.machine_name}",
             f"--directory={self.root}",
             f"--bind={libdir}/osbuild-run:/run/osbuild/osbuild-run",
-            *[f"--bind={b}" for b in (binds or [])],
+            f"--bind-ro={libdir}/osbuild:/run/osbuild/osbuild",
```

Comment:
The fact that we have `osbuild/osbuild` here points to something awkward. Not sure what the solution is, yet.

Reply:
Yeah, it is a bit odd. Let's get back to that.